frontend/packages/console-plugin-sdk/src/typings/features.ts

```diff
@@ -6,7 +6,7 @@ import { Action } from 'typesafe-actions';
 namespace ExtensionProperties {
   interface FeatureFlag {
     /** The name of the feature flag. */
-    flag: string;
+    flag: string | string[];
```

Comment:
@vojtechszocs I could make this an array by default and change all the usages of it in plugin files but IDK how useful that will be. Suggestions...?

Reply:
Type unions tend to complicate code that interprets their values. Therefore, if we need to support an array of values, I'd prefer `string[]` instead of `string | string[]`.

**Design question:** each `*FeatureFlag` extension type was meant to add a _single new_ feature flag, hence the singular form in `flag: string`.

What's the intended design here?

- Are we trying to gate `*FeatureFlag` extensions themselves? (Which goes against the current design since they extend `AlwaysOnExtension`.)
- Are we trying to add multiple feature flags with different names but using the same value resolution? (Which doesn't seem too useful to me.)